sdk/servicebus/service-bus/src/receivers/receiverCommon.ts

```diff
@@ -155,28 +168,47 @@ export function deadLetterMessage(
   const dispositionStatusOptions: DispositionStatusOptions = {
     propertiesToModify: actualPropertiesToModify,
     deadLetterReason: propertiesToModify?.deadLetterReason,
-    deadLetterDescription: propertiesToModify?.deadLetterErrorDescription
+    deadLetterDescription: propertiesToModify?.deadLetterErrorDescription,
+    retryOptions
   };
 
-  return settleMessage(
-    message,
-    DispositionType.deadletter,
-    context,
-    entityPath,
-    dispositionStatusOptions
-  );
+  return settleMessage(message, DispositionType.deadletter, context, entityPath, {
+    ...dispositionStatusOptions,
+    retryOptions
```

Comment:
`retryOptions` is already part of `dispositionStatusOptions` from a few lines above. So, why pass it again here?

Reply:
Ha! Overzealous consistency! Removed.